rpcserver.go

```diff
@@ -6801,6 +6815,13 @@ func (r *rpcServer) DecDisplayForAssetID(ctx context.Context,
 			"for asset_id=%v :%v", id, err)
 	}
 
+	return getDecimalDisplayNonStrict(meta)
+}
+
+// getDecimalDisplayNonStrict attempts to decode a decimal display value from
+// metadata. If no custom decimal display value is decoded, the default value of
+// 0 is returned without error.
+func getDecimalDisplayNonStrict(meta *proof.MetaReveal) (uint32, error) {
```

Comment:
I think this should return an `fn.Option` instead of defaulting to 0. Later at the callsite, `taprpc.MarshalAsset` takes an option anyway.

Reply:
That's added in a later commit. Perhaps you were looking at an outdated version?

Reply:
I don't think I was looking at an outdated version of the commit set. I was going through each commit in order. Later commits change the changes of earlier commits.